idea-plugin/src/main/kotlin/arrow/meta/ide/dsl/editor/lineMarker/LineMarkerSyntax.kt

```diff
@@ -236,7 +238,12 @@ interface LineMarkerSyntax {
     clickAction: AnAction? = null
   ): ExtensionPhase =
     addLineMarkerProvider(
-      { transform(it)?.identifyingElement },
+      {
+        transform(it)?.let { psi ->
```

Comment:
the registered element here needs to be set by the user. 
That is a wrong assumption that it is either this or the other way with LeafPsiElement

Reply:
Line markers should target leaf psi elements.. any help to do it differently is welcome!